### DISCUSSION TITLE
Intellij IDEA does not stay in fullscreen mode

### DISCUSSION THREAD

**Discussion by RiccardoBarbieri** [i3/i3#5701] [Q&A] [answered]
Hi, I have this problem that when I put intellij IDEA in fullscreen mode it almost instantly exit that mode. It started recently and I cannot identify a change that could have started this issue.
How could I begin to debug this issue?

**Comment by okraits** [comment]
I would execute Intellij IDEA from a terminal, possibly with verbose output, enable fullscreen mode and watch for helpful output in the terminal.

**Reply by RiccardoBarbieri** [reply]
Thanks for the suggestion but I already tried that, without much luck.

**Comment by RiccardoBarbieri** [comment, marked as answer]
Same issue documented at #5695, closing
